addons/default/visiosoft/advs-module/src/Http/Controller/CategoriesController.php

```diff
@@ -8,12 +8,24 @@ class CategoriesController extends PublicController {
 
     public function listByCat($cat) {
         $advs = DB::table('advs_advs')
-            ->where('category_id', $cat)
+            ->where('cat1', $cat)
```

Comment:
do we need this? @Diatrex

Reply:
It was not working before because we don't have 'category_id' column in advs model, I just fixed it. We can remove it if you forgot about it. @fatihalp

Reply:
if you tested it's ok then @spektra2147 check it

Reply:
yes I tested it, it is working.